thumbor/handlers/__init__.py

```diff
@@ -485,57 +485,45 @@ def _fetch(self, url):
         :rtype:
         """
         fetch_result = FetchResult()
+        loader_result = None
 
         storage = self.context.modules.storage
 
         yield self.acquire_url_lock(url)
 
         try:
             fetch_result.buffer = yield gen.maybe_future(storage.get(url))
-            mime = None
 
             if fetch_result.buffer is not None:
                 self.release_url_lock(url)
```

Comment:
is this for speed to release the url  a little bit earlier than inside the `finally:` block? Or why is this still here? (not sure if the `metrics.incr` is so slow that this duplication of code is still worth it)